apps/comps/components/agent-profile/comps-table.tsx

```diff
@@ -46,58 +46,53 @@ export function CompetitionTable({
 
   return (
     <>
-      <div className="overflow-x-auto rounded border">
+      <div className="overflow-x-auto rounded">
         <Table className="min-w-[800px]">
-          <TableHeader className="text-muted-foreground bg-gray-900 font-mono text-xs">
+          <TableHeader>
```

Comment:
the  bg/color arent used on other tables; we should keep this consistent.

also, this was refactored so that the font weight & mono are applied in the `ui2` package. we no longer need to apply them on the TableHeader or header definitions.